### DISCUSSION TITLE
accessible-like components

### DISCUSSION THREAD

**Discussion by Bad3r** [ElianCodes/brutal#7] [Ideas]
Hello!
I just came across this theme, and I am in love.
It looks great, and I am considering it for my upcoming blog.

I am wondering if it would be possible to add components like the ones in the Accessible theme: https://www.accessible-astro.dev/accessible-components/

**Comment by ElianCodes** [comment]
In my opinion, this should be possible for sure! You can of course add your own NPM packages to the theme without any problems.

Or are you suggesting to add more default components to the theme? If this is the case, I've been thinking if I should release a UI package called `brutal-ui`, which would make components more generic to implement.

What do you think?

**Reply by Bad3r** [reply]
> Or are you suggesting to add more default components to the theme? If this is the case, I've been thinking if I should release a UI package called `brutal-ui`, which would make components more generic to implement.
> 
> 
> 
> What do you think?

Yes I am suggesting adding more default components in Brutal style. A separate package or adding more components directly would work.

**Reply by ElianCodes** [reply]
Allright! I'll take a look into that! Thanks for your valuable feedback!

**Comment by ElianCodes** [comment]
I'm building this out in <https://github.com/ElianCodes/brutal-ui>. it's still a work in progress, but I'll be adding components gradually and will keep improving them!

**Reply by Bad3r** [reply]
Awesome, will check it out! :star2: 
Btw, I recently came across Figma's website image hover effect:
![figma-effect](https://user-images.githubusercontent.com/25513724/226194743-c48e7512-9aa5-44c8-99a1-994b5d9b3d0a.gif)
https://help.figma.com/hc/en-us/articles/360041949073-Use-Chrome-Developer-Tools

I feel like this effect would fit great in Brutal-ui for either the blocks on the homepage or images, etc.. 

What do you think?

**Reply by ElianCodes** [reply]
Love it! I'll look into it for sure!
